app/main.py

```diff
@@ -1,3 +1,59 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+
+        if isinstance(other, Distance):
+            return Distance(other.km + self.km)
+        return Distance(other + self.km)
```

Comment:
It is a better practise to place an instance attribute in the first place and the upcoming argument in the second position during different operations.

Reply:
If you swap places, the code doesn’t work, I don’t understand why?